SplunkRumWorkspace/SplunkRum/SplunkRum/AppLifecycle.swift

```diff
@@ -96,7 +96,7 @@ func lifecycleEvent(_ event: String) {
     if event == UI_APPLICATION_WILL_TERMINATE_NOTIFICATION {
         let now = Date()
         let span = buildTracer().spanBuilder(spanName: "AppTerminating").setStartTime(time: now).startSpan()
-        span.setAttribute(key: "component", value: "AppLifecycle")
+        span.setAttribute(key: "component", value: "app-lifecycle")
```

Comment:
above on line 78, this is set as "app-lifecycle". I thought they should match. Given the Spans test to be capitalized camel case, I used the style from line 78 for the component.